.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
you've checked in a .DS_Store which is a mac system file. Those should be included in your .gitignore. Add .DS_Store and delete the file